pkg/storage/system_intake.go

```diff
@@ -275,20 +275,34 @@ func (s *Store) FetchSystemIntakes(ctx context.Context) (models.SystemIntakes, e
 
 // FetchSystemIntakesByStatuses queries the DB for all system intakes matching a status filter
 func (s *Store) FetchSystemIntakesByStatuses(ctx context.Context, allowedStatuses []models.SystemIntakeStatus) (models.SystemIntakes, error) {
-	intakes := []models.SystemIntake{}
-	const byStatusClause = `
-		WHERE system_intakes.status IN (?)
+	var intakes models.SystemIntakes
+	query := `
+		SELECT
+			system_intakes.*,
+			business_cases.id as business_case_id,
+			intakes_and_notes.content AS last_admin_note_content,
+			intakes_and_notes.created_at AS last_admin_note_created_at
+		FROM
+			(	SELECT
+					distinct ON (system_intakes.id) system_intakes.id, notes.content, notes.created_at
+				FROM system_intakes
+					LEFT JOIN notes on notes.system_intake = system_intakes.id
+				WHERE system_intakes.status IN (?)
+				ORDER BY system_intakes.id, notes.created_at DESC
+			) AS intakes_and_notes
+		LEFT JOIN system_intakes ON system_intakes.id = intakes_and_notes.id
+		LEFT JOIN business_cases ON business_cases.system_intake = system_intakes.id
```

Comment:
This is using a subquery to get the ID and the related note fields for any matching system intakes, and then using the result of that query to collect all of the system intake data and the business case ID.

There are a few different ways to write this query, but I find this easier to reason about than using `GROUP BY` and a lot of `JOIN`s.